soh/soh/GameMenuBar.cpp

```diff
@@ -338,12 +338,12 @@ namespace GameMenuBar {
             {
                 if (ImGui::BeginMenu("Time Savers"))
                 {
-                    UIWidgets::PaddedEnhancementSliderInt("Text Speed: %dx", "##TEXTSPEED", "gTextSpeed", 1, 5, "", 1, false, false, true);
-                    UIWidgets::PaddedEnhancementSliderInt("King Zora Speed: %dx", "##MWEEPSPEED", "gMweepSpeed", 1, 5, "", 1, false, false, true);
-                    UIWidgets::EnhancementSliderInt("Biggoron Forge Time: %d days", "##FORGETIME", "gForgeTime", 0, 3, "", 3);
+                    UIWidgets::PaddedEnhancementSliderInt("Text Speed: %dx", "##TEXTSPEED", "gTextSpeed", 1, 5, "", 1, true, false, true);
+                    UIWidgets::PaddedEnhancementSliderInt("King Zora Speed: %dx", "##MWEEPSPEED", "gMweepSpeed", 1, 5, "", 1, true, false, true);
+                    UIWidgets::PaddedEnhancementSliderInt("Biggoron Forge Time: %d days", "##FORGETIME", "gForgeTime", 0, 3, "", 3, true, false, true);
```

Comment:
another one where padding is added, another before/after shot would be nice

Reply:
Before:
![image](https://user-images.githubusercontent.com/30329717/222878095-309b4a06-6e8e-4603-981c-ebc8413c6d2c.png)

Reply:
After:
![image](https://user-images.githubusercontent.com/30329717/222878149-bab1e16d-1016-435a-8b1e-581573a26125.png)

Reply:
You'll see that there's more space between "Vine/Ladder" and the slider above it, so it's now consistent with the spacing between all the other sliders.  You'll also see that I fixed the extra space after the "Link as default file name" checkbox.